FirelandsProject/firelands-cata
# (Core/Apps) Restructuring auth and world files

### Changes proposed

- In order to find a solution to the problem we are having with the Windows connection, and reviewing the files to find the error, we are working to rearrange the files to make them more readable.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10
- Github

### How to Test the Changes:


1. Actually, there is not much to test, just check the files before and after the change, right here in github.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
